server/halomod_app/__init__.py

```diff
@@ -409,4 +395,16 @@ def ascii():
 
         return send_file(buff, as_attachment=True, attachment_filename="all_plots.zip")
 
+    backend_constants = utils.generate_constants()
+
+    @app.route('/constants', methods=["GET"], strict_slashes=False)
```

Comment:
So why do we need a specific endpoint for this? Why can't it be generated as part of the deployment/run process? 
Only the client needs it so we can create a script that's called inside package.json that generates backend_constants and saves it in the correct location.

Reply:
Would this run on server start?

Reply:
How would it access the python packages on the client side?

Reply:
No run on the CI, using an sh file called from npm or elsewhere